### PR TITLE
ISAICP-5568: Improve user profile layout v2.

### PR DESCRIPTION
**Pull request by papegaill**

### CODE REVIEW

File: web/themes/joinup/templates/user/user--group-header.html.twig [hunk 1]
```diff
@@ -26,49 +26,69 @@
...
+                        <div>ICT</div>
+                        <div>Science and technology</div>
+                        <div>Collaboration</div>
+                        <div>Free and open source software in Public Administrations</div>
+                        <div>ICT standardisation in eGovernment</div>
+                        <div>Open government</div>
+                        <div>eGovernmentttt</div>
+                    </div>
+                  </span>
+                </div>
```
Comment (pfrenssen) [inline web/themes/joinup/templates/user/user--group-header.html.twig:86]: This test data should be removed.

File: web/themes/joinup/templates/user/user--group-header.html.twig [hunk 2]
```diff
@@ -26,49 +26,69 @@
...
+            <div class="mdl-grid mdl-grid--no-spacing">
+
+              <div class="mdl-cell mdl-cell--6-col mdl-cell--4-col-wide">
+                {% if content.field_user_organisation|render %}
+                  <div class="stats__wrapper stats__wrapper-organisation">
+                    <span class="stats__text bold">{{ 'Organisation'|t }}:</span>
+                    <span class="stats__text">{{ content.field_user_organisation }}</span>
+                  </div>
+                {% endif %}
+              </div>
```
Comment (pfrenssen) [inline web/themes/joinup/templates/user/user--group-header.html.twig:63]: If the user doesn't enter an organisation then there is a strange void left in place here. We should probably not render the empty div in this case.

2020-04-06-115304_7680x2160_scrot

**Comment by pfrenssen** [comment]
Looking good overall, found two small issues. We are also removing the country of the user but according to functional requirements this should be visible. Let's wait for an answer from the functional team on the JIRA issue before continuing with this.

File: web/themes/joinup/templates/user/user--group-header.html.twig (hunk 1 above)
Comment (papegaill) [reply web/themes/joinup/templates/user/user--group-header.html.twig]: Oups, updated ;)

File: web/themes/joinup/templates/user/user--group-header.html.twig (hunk 2 above)
Comment (papegaill) [reply web/themes/joinup/templates/user/user--group-header.html.twig]: Thank you for this feedback,PR updated accordingly.

**Comment by papegaill** [comment]
Hi @pfrenssen thank you for your review, I updated the PR based on your remarks and I also added 'members of' and 'country ' fields as requested.

**Comment by papegaill** [comment]
Hi @claudiu-cristea many thanks for review and feedback, I updated the PR accordingly.

**Comment by claudiu-cristea** [comment]
@papegaill, excellent, thank you. In fact the failed test, showed that the label has been showed without having a field value. I've wrongly changed the test. Now I reverted the test change but pushed with `--force`, so you'll delete the local branch and checkout again from remote.

We're almost there. However...

> (...) Also the country value should be inline with the label, as this is a single value field

...this was not implemented.

**Comment by papegaill** [comment]
@claudiu-cristea Thank you, Added css fixe to have country value inline with the label.

**Pull request merged**